src/components/AppDrawer/AppDrawer.js

```diff
@@ -205,12 +197,14 @@ function AppDrawer({ forceRerender, setForceRerender }) {
               'lessonProgressPercent',
               data?.result?.result?.progress || 0
             );
+            handleNavigate(data?.result?.result?.milestone || 'discoverylist')
+            fetchDataFromApi(value)
```

Comment:
After navigating - fatch API is required?

Reply:
it's a function name 
we are getting points as per current language